packages/apps/esm-login-app/src/location-picker/location-picker.component.tsx

```diff
@@ -103,30 +113,30 @@ const LocationPicker: React.FC<LocationPickerProps> = ({ hideWelcomeMessage, cur
 
   // Handle cases where the location picker is disabled, there is only one location, or there are no locations.
   useEffect(() => {
-    if (!isLoading && !searchTerm) {
+    if (!isLoadingLocations && !debouncedSearchQuery) {
```

Comment:
`!searchTerm` seems more correct here, no?

```suggestion
    if (!isLoadingLocations && !searchTerm) {
```